test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -473,7 +474,7 @@ TEST_F(AWSLambdaFilterTest, ALBDecodingBasic) {
 
   filter_->setEncoderFilterCallbacks(filter_encode_callbacks_);
   auto res = filter_->encodeHeaders(response_headers, true);
-  EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, res);
+  EXPECT_EQ(Http::FilterHeadersStatus::Continue, res);
```

Comment:
Yuval called out a preexisting bug in the lambda filter in this comment: https://github.com/solo-io/envoy-gloo/pull/167#discussion_r922346251

Essentially, we shouldn't stop iteration if end_stream (the second argument in the call to `encodeHeaders` is `true`)

I fixed that here: https://github.com/solo-io/envoy-gloo/commit/56486f510d7c62efe8ba72209e1c25bbb23a3462

And now the expected value of `res` changes, because `end_stream` is `true` in the call to `encodeHeaders`